packages/swap/src/event-handlers/networkBroadcastSuccess.ts

```diff
@@ -14,7 +14,7 @@ export const networkBroadcastSuccessArgs = (chain: Chain) =>
           .transform((v) => `${v.blockNumber}-${v.extrinsicIndex}`),
         hexString,
       ])
-      .transform((v) => (chain === 'Bitcoin' ? reverseBytes(v) : v))
+      .transform((v) => (chain === 'Bitcoin' ? reverseBytes(v.substring(2)) : v))
```

Comment:
we exposed the bitcoin tx hash with a leading `0x` till now (bitcoin explorers do not support that when i tested) 😕 will add a migration for the existing broadcasts